scripts/pylib/twister/twisterlib/handlers.py

```diff
@@ -326,10 +326,6 @@ def handle(self, harness):
 
         handler_time = time.time() - start_time
 
-        if self.options.coverage:
-            subprocess.call(["GCOV_PREFIX=" + self.build_dir,
-                             "gcov", self.sourcedir, "-b", "-s", self.build_dir], shell=True)
```

Comment:
with `shell=True` and list of args this call did only GCOV_PREFIX assignment when twister runs on linux.